src/main/scala/io/shiftleft/js2cpg/parser/JavaScriptParser.scala

```diff
@@ -12,9 +12,9 @@ object JavaScriptParser {
 
   private val logger = LoggerFactory.getLogger(JavaScriptParser.getClass)
 
-  // 11 is internally used by graaljs for the ECMA script version 2020.
+  // A value >= 13 is internally used by graaljs for the ECMA script version 2022.
   // Sadly the do not provide a define for this.
-  private val ecmaVersion2020 = 11
+  private val ecmaVersion = 13
```

Comment:
Update parser to ECMA script version 2022 to support latest async/await constructs.
It's only the parser - does not affect transpilation.